pulp_ansible/app/tasks/copy.py

```diff
@@ -0,0 +1,92 @@
+from django.db import transaction
+
+
+@transaction.atomic
+def copy_content(config, dependency_solving):
+    """
+    Copy content from one repo to another.
+    Args:
+        source_repo_version_pk: repository version primary key to copy units from
+        dest_repo_pk: repository primary key to copy units into
+        criteria: a dict that maps type to a list of criteria to filter content by. Note that this
+            criteria MUST be validated before being passed to this task.
+        content_pks: a list of content pks to copy from source to destination
+    """
+    def process_entry(entry):
+        source_repo_version = RepositoryVersion.objects.get(pk=entry["source_repo_version"])
```

Comment:
Doesn't seem these models are imported anywhere.

Reply:
This code is still in development. I need to push latest.

Reply:
pushed.